addons/point_of_sale/static/tests/tours/utils/common.js

```diff
@@ -35,7 +35,7 @@ export function refresh() {
     };
 }
 
-export function elementDoesNotExist(selector) {
+export function isElementDoesNotExist(selector) {
```

Comment:
this phrasing is weird...
`elementDoesNotExist` makes sense; the new option does not